site/content/entry/yuzu-progress-report-jan-2022/index.md

```diff
@@ -0,0 +1,366 @@
+
+german77 also went ahead and implemented the graphical elements needed to [report the battery levels](https://github.com/yuzu-emu/yuzu/pull/7735) of UDP controllers.
+While this code has been in yuzu's source for some time already, preparing the appropriate front-end graphics and the subsequent implementation into the UI had been pending tasks for a long time.
+
+{{< imgs
+	"./battery.png| Pretty useful for Joy-Cons!"
+  >}}
+
+While [updating the AMD FidelityFX Super Resolution](https://github.com/yuzu-emu/yuzu/pull/7768) (`FSR`) dependency to the latest version, [Moonlacer](https://github.com/Moonlacer) changed the text string to replace the brackets around the `Vulkan Only` message with parenthesis, for consistency with all the other text in our interface.
+
+On a similar vein, [gidoly](https://github.com/gidoly) corrected a series of spelling in the strings describing the name of commercial gamepads, namely the [PlayStation](https://github.com/yuzu-emu/yuzu/pull/7713) and [Xbox](https://github.com/yuzu-emu/yuzu/pull/7715) controllers.
+
+Based on a patreon poll conducted in our discord server, gidoly also made the necessary changes to make the `Dark Colourful` theme the [default scheme](https://github.com/yuzu-emu/yuzu/pull/7719) used when running yuzu for the first time on Windows.
+
+As a follow-up, [v1993](https://github.com/v1993) also made changes so that the colourful scheme (i.e. dark or light) used for the first time [depends on the system-wide theme](https://github.com/yuzu-emu/yuzu/pull/7755) in *NIX systems.
+
+{{< imgs
+	"./theme.png| Default by popular demand"
+  >}}
+
+Naturally, users can still change the theme through the configuration settings, if so desired.
+
+## Input changes
+
+[german77](https://github.com/german77) has been *specially* busy this month, so there’s quite a bit to cover in this section.
+
+Users reported that the game `パワプロクンポケットR` and other games of the Power Pocket saga crashed in-game.
+This is caused by the `SetNpadAnalogStickUseCenterClamp` service being able to initialise the applet_resource subsystem even if it wasn’t initialised before.
+[Emulating this behaviour solves the issue and makes some game modes playable](https://github.com/yuzu-emu/yuzu/pull/7726). 
+Some game modes show that yuzu lacks support for some vertex formats, causing crashes.
+
+{{< imgs
+	"./power.png| パワプロクンポケットR"
+  >}}
+
+[Only supported controller types will be shown in the applet now](https://github.com/yuzu-emu/yuzu/pull/7663).
+This depends on what each game informs as compatible.
+
+`Fullkey` is the codename used by Nintendo to refer to a generic type of controller that informs itself as a Pro Controller if you connect an “unsupported” gamepad, this can refer to the GameCube, NES, SNES, N64 and Sega Genesis controllers.
+In case of problems, the console fallbacks those types to a Pro Controller, german77 [added this functionality to yuzu too](https://github.com/yuzu-emu/yuzu/pull/7664).
```

Comment:
```suggestion
`Fullkey` is the codename used by Nintendo to refer to a generic type of controller that reports itself as a Pro Controller if you connect an “unsupported” gamepad. This can refer to the GameCube, NES, SNES, N64 and Sega Genesis controllers.
In case of problems, the console falls back to reporting a Pro Controller for those types of controllers. german77 [added this functionality to yuzu as well](https://github.com/yuzu-emu/yuzu/pull/7664).
```